client/data/authorizations/selectors.ts

```diff
@@ -18,24 +28,49 @@ import {
  *
  * @return {Object} The list of authorizations for the given query.
  */
-const getAuthorizationsForQuery = ( state: Record< string, any > ) => {
-	return state.authorizations;
+const getAuthorizationsForQuery = (
+	state: Record< string, any >,
+	query: Query
+) => {
+	const index = getResourceId( query );
```

Comment:
Thinking out loud - using a stringified query string as index makes it difficult to visualize how the state is maintained. Would having a simpler index be an oversimplication of the underlying logic?

Reply:
This will be used when we have pagination in place. If you check the transactions state for instance, you'll see that the state has this shape:

<img width="620" alt="Screen Shot 2022-09-08 at 18 53 41" src="https://user-images.githubusercontent.com/1553182/189180931-ae1abc84-f31a-4e28-8571-820d1a29e831.png">

We will have something similar for authorizations when pagination is implemented. I'd prefer to keep it as it is since we will use it eventually, but I am also happy to change it if you think we should. I wonder what would be a better index in this case?

Reply:
Sounds good, thanks for explaining :) We can keep it like this.